docs/website/docs/building-from-source/getting-started.md

```diff
@@ -45,7 +45,7 @@ git submodule update --init
 Configure then build all targets using CMake:
 
 ``` shell
-cmake -B ../iree-build/ .
+cmake -B ../iree-build/ -DCMAKE_BUILD_TYPE=RelWithDebInfo .
```

Comment:
I feel like this needs a tip below which says that this disables various debugging features (`-debug`, asserts) and if those are valuable, building with `Debug` is recommended, at the expense of the resulting compiler and runtime being significantly slower.

(really wish we could enable those in release mode, so maybe can update this guidance at some point)

Reply:
SG. Added a tip:

```
???+ Tip
    We recommend using the `RelWithDebInfo` build type by default for a good
    balance of debugging information and performance. The `Debug`, `Release`,
    and `MinSizeRel` build types are useful in more specific scenarios. See the
    [official CMake documentation](https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html)
    for more details.
```

Reply:
Possibly also "note that several useful LLVM debugging features are only available in Debug builds." Nothing like expecting to find certain features and not knowing how to get them (ie. `-debug`)